rfcs/convergence/ending-ie11-support.md

```diff
@@ -0,0 +1,70 @@
+
+### Keep calm and carry on
+
+Our partners and products in M365 will either adhere to the August 2021 deadline or receive a 6 month extension for IE11 support.
+Since Fluent v9 is still in alpha release, and is planned to be at beta at August 2021, we will be ready for early adopters.
+Until that deadline, users that require full support for IE11 can do so with the existing versions of `@fluentui/react` and
+`@fluentui/react-northstar` that do provide support.
+
+## Motivation
+
+### Reduced transpilation - (perf + bundlesize improvements)
+
+The possibility to use native ES2015/ES6 features without the risks to breaking browser compatibility, we can reduce the
+transpilation of IE11 features that results in more code. This can result in perf and bundle size gains, and tslib
+will no longer be needed as a hard dependency to emulate the missing features of modern javascript.
+
+### Improved Theming
+
+Theming a web application is always a challenge and can easily lead to performance issues. Performance has been a challenge for the current
+versions of fluent that evaluated theme values with javascript. Fluent v9 uses CSS variables for its theming and styling solution. This means
+that it does not need to have a different set of classes for each theme and css rules never change regardless of theme which is a
+big boost to performance.
+
+Our future theming system also uses the [Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)
+in a key step to create the CSS variables that map to theme values. Without this feature that IE11 lacks, we would need
+to dedicate more time to another solution for this.
+
+### Maintenance Burden
+
+The lack of support that IE11 offers for modern (ES2015) javascript has not stopped developers from using it. With EOL of IE11
+we no longer have to consider the language features used across the entire codebase, and figure out the the real support our
+transpiled code has for the browser. Since dropping support of IE11 is a breaking change, once we support it in a major version
+of Fluent, we can't drop support until the next major release.
+
+Our tooling will improve and its complexity will decrease. With babel and `preset-env` will be able to use the same tooling
+in our tests, storybook and the code we ship. Our build times and and test times will also decrease which means faster CI
+times overall.
+
+### Motivating our partners
+
+There are clear benefits we have planned with Fluent v9, from easier composition to performant styling. We want our partners
+to leverage the full set of advantages that Fluent v9 has to offer. The web ecosystem needs to move forward and supporting
+IE11 will inevitably make it slower. By taking this stance in Fluent v9 we give a clear motivation to our partners to
+modernize their own products.
+
+## Pros and Cons
+
+We understand that this might cause initial adoption headaches from our partners. However, they should also generally be targeting
```

Comment:
so which sentence are pros and cons ?

Reply:
I'll just rename this section to `Cons`, the pros have been enumerated already in the previous section

Reply:
sounds good. I'd still maintain consistency. 

something like:

```
### Pros

- already mentioned in previous chapters

### Cons

- ............
- ............
```